docs/source/containerization/kubernetes.mdx

```diff
@@ -1,264 +1,250 @@
 ---
-title: Kubernetes and the router
-description: Using router images with kubernetes
+title: Deploy in Kubernetes
+description: Self-hosted deployment of Apollo Router in Kubernetes
 ---
 
 import { Link } from 'gatsby';
+import ElasticNotice from '../../shared/elastic-notice.mdx';
+import HelmShowOutput from '../../shared/helm-show-router-output.mdx';
+import K8SManualConfig from '../../shared/k8s-manual-config.mdx';
+import RouterCommonConfig from '../../shared/router-common-config.mdx';
+import CoprocTypicalConfig from '../../shared/coproc-typical-config.mdx';
+
+Learn how to deploy the Apollo Router, for a self-hosted supergraph with GraphOS, in Kubernetes by using Helm charts.
+
+```mermaid
+flowchart LR;
+  clients(Clients);
+  subgraph "Kubernetes cluster";
+  proxy(["<b>Proxy</b>"])
+  router_a(["<b>Apollo Router</b>"]);
+  coproc_a(["<b>Coprocessor</b>"]);
+  router_b(["<b>Apollo Router</b>"]);
+  coproc_b(["<b>Coprocessor</b>"]);
+  router_a <-.-> coproc_a;
+  router_b <-.-> coproc_b;
+  end;
+  clients -.->|Query| proxy;
+  proxy -.-> router_a & router_b;
+  class clients secondary;
+```
+
+The following guides provide you the steps to:
+
+* Get a router Helm chart from the Apollo container repository.
+* Deploy a router with a basic Helm chart.
+* Configure chart values to export metrics, enable Rhai scripting, and deploy a coprocessor.
+* Choose chart values that best migrate from a gateway to the router.
+
+<ElasticNotice />
 
-## Sample Kubernetes Configuration
+## About the router Helm chart
 
-> Note: The Apollo Router is made available under the Elastic License v2.0 (ELv2).  This applies to its source code and all distributions, including versions installed via Helm charts.  Read [our licensing page](https://www.apollographql.com/docs/resources/elastic-license-v2-faq/) for more details.
+[Helm](https://helm.sh) is a package manager for Kubernetes (k8s). Apollo provides an application Helm chart with each release of Apollo Router in GitHub. Since the router version 0.14.0, Apollo has released the router Helm chart as an [Open Container Initiative (OCI)](https://helm.sh/docs/topics/registries/) image in our GitHub container registry.
```

Comment:
It's an unfortunate name collision, as GitHub explicitly names it its ["Container registry"](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry). We need an improvement to Apollopedia, to ignore specified instances like this.

Reply:
Ah I see, I think it's referred to as "container repository" in a couple of places in different pages in this PR, so it might be worth doing a find/replace.